src/services/TeamService.js

```diff
@@ -253,8 +253,8 @@ getTeam.schema = Joi.object().keys({
  * @returns the team job
  */
 async function getTeamJob (currentUser, id, jobId) {
-  if (currentUser.isMachine) {
-    const m2mToken = await helper.getM2Mtoken()
```

Comment:
Replace `helper.getM2Mtoken()` with `helper.getTopcoderM2MToken()` is needed, otherwise it would get 403 error from `GET /projects/:projectid`.